CONTRIBUTING.md

```diff
@@ -66,110 +62,133 @@ info
 
 ```sh
 npm run create:component -- ${Component Name}
 ```
 
-But apart from adding a new component, there are some guidelines which we want to follow.
+### Does It Belong in UI Kit?
 
-While creating a new component there are a couple of steps everyone should think about before releasing a new component. This is important because once we released a component and it already got adopted it's rather difficult to remove or change the functionality of a component.
-
-### Rigid VS. Flexible
-
-Generally, a component should rather be rigid than flexible. That means that we rather create more components then creating one which has a high complexity. Although a flexible component has a higher rate of reusability in more contexts the complexity increases a lot more. Which is an easy way to introduce a lot of errors to a component, that is why we prefer to have rigid components. A rigid component is simple, easy to understand, easy to test, less code and also improves the consistency.
+Before adding a new component to UI Kit, we need to carefully consider whether it belongs in UI Kit. If we're too quick to add new components to UI Kit, we will end up bloating our component library and spend too much time maintaining reuasble components.
 
-> It's always easy to add flexibility later, it's nearly impossible to remove flexibility.
-
-> -- Random Person on the Internet
-
-While creating a component think about one single reappearing use-case and create a component to solve that use-case. And do not try to think about a million use-cases which one component could handle. Fewer features mean more simplicity and consistency.
+When a new component is being created, work with the design team to document design and usage guidelines in the component folder's README.md file.
 
-### Introducing 3rd Party Components
+#### Consider the Following Before Adding a New Component
 
-There are 3 ways to introduce 3rd party components to the `UI-kit`
+**Is it likely this component be used in more than 1 app?**
+For example, a component for a card that may contain any content would be a good candidate for ui-kit. A card that specifically renders data specific to an object in an napp is not.
 
-* Link  
-  We add a link to that component to the documentation. This is not a favorable way.
-* Wrap  
-  We are creating our own component but in that component we are utilising a 3rd party component, this solution is the favourable solution for the `UI-kit` as with that solution we are adding consistency to the 3rd party components and there is a single point to change the usage of that component if we decide to update the component or replace the component with our own implementation or the other implementation.
-* Fork  
-   We should do that as a last resort if we do not see any other solution.
+**Does it fit in with a "theme" of other components in the UI Kit?**
+For example, if we have a toast and an inline alert box, a page banner would make sense to exist in the ui-kit
```

Comment:
i have to admit that i don't immediately see the correlation of `toast`, `alert box` and `page banner`. 🤔 

but that's in no way blocking....